docs/walkthrough.md

```diff
@@ -50,132 +51,92 @@ When we create this `ClusterServiceBroker` resource, the service catalog control
 by querying the broker server to see what services it offers and creates a
 `ClusterServiceClass` for each.
 
-We can check the status of the broker using `kubectl get`:
+We can check the status of the broker using `svcat describe`:
 
 ```console
-$ kubectl get clusterservicebrokers ups-broker -o yaml
-apiVersion: servicecatalog.k8s.io/v1beta1
-kind: ClusterServiceBroker
-metadata:
-  creationTimestamp: 2017-11-01T14:11:29Z
-  finalizers:
-  - kubernetes-incubator/service-catalog
-  generation: 1
-  name: ups-broker
-  resourceVersion: "6"
-  selfLink: /apis/servicecatalog.k8s.io/v1beta1/clusterservicebrokers/ups-broker
-  uid: 8df4e501-bf0e-11e7-9e29-0242ac110004
-spec:
-  relistBehavior: Duration
-  relistDuration: 15m0s
-  relistRequests: 0
-  url: http://ups-broker-ups-broker.ups-broker.svc.cluster.local
-status:
-  conditions:
-  - lastTransitionTime: 2017-11-01T14:12:30Z
-    message: Successfully fetched catalog entries from broker.
-    reason: FetchedCatalog
-    status: "True"
-    type: Ready
-  reconciledGeneration: 1
+$ svcat describe broker ups-broker
+  Name:     ups-broker
+  URL:      http://ups-broker-ups-broker.ups-broker.svc.cluster.local
+  Status:   Ready - Successfully fetched catalog entries from broker @ 2018-03-02 16:03:52 +0000 UTC
 ```
 
-Notice that the `status` field has been set to reflect that the broker server's
+Notice that the status reflects that the broker's
 catalog of service offerings has been successfully added to our cluster's
 service catalog.
 
-# Step 3 - Viewing `ClusterServiceClass`es and `ClusterServicePlan`s
+# Step 3 - Viewing ClusterServiceClasses and ClusterServicePlans
 
 The controller created a `ClusterServiceClass` for each service that the UPS broker
-provides. We can view the `ClusterServiceClass` resources available in the cluster by
-executing:
+provides. We can view the `ClusterServiceClass` resources available by executing:
 
 ```console
-$ kubectl get clusterserviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
```

Comment:
same thing here (and elsewhere), let's keep the kubectl stuff in and add svcat, since users are probably most familiar with that tool in a vacuum.